postgres.pl

```diff
@@ -88,6 +88,15 @@
     },
 );
 
+# New configopts per version
+my %new_configopts_per_version=(
+        '11' => ['--with-llvm'],
+        'dev' => ['--with-llvm']);
+
+# No idea what version it could be, but lets's do this right now
+my %deprecated_configopts_per_version=(
+        );
```

Comment:
`--with-wal-segsize` has been removed in 11/dev, we could add it to test this one